### ISSUE TITLE
Getters for JsonPatch* class attributes

### ISSUE THREAD

**Issue by ssteveli** [fge/json-patch#33]
Was there a particular design reason no getters were provided in the JsonPatch classes?  I'm deserializing json over the wire into instances of these classes and attempting to provide for a JSR303 validation implementation allowing me to limit patch operations, without public methods to access class attributes (i.e. path), it's seemingly impossible.  I'd hate to convert to something that I can access the attributes, it seems cumbersome.

Thoughts?

**Comment by box-jhuffaker** [comment]
I can't talk to @fge's rationale for it.  fge hasn't really been responding here.  I'm not sure if there is a new maintainer.  If you made a pull request, I could find some time to setup a build on box-metadata/json-patch and publish a build of it to maven central.

**Comment by chb0github** [comment]
Time for a fork?

**Comment by huggsboson** [comment]
I got the builds setup and am happy to accept pull requests at the [crate-metadata/json-patch](https://github.com/crate-metadata/json-patch) fork and build them to maven central @ net.box.json-patch.

**Comment by chb0github** [comment]
Can you do an actual build and release so I can depend on it?

**Comment by chb0github** [comment]
Can you do an actual build and release so I can depend on it?

**Comment by huggsboson** [comment]
yeah I can do one at net.box#json-patch instead of com.github.fge.json-patch.  We have to get the change merged over at the fork first though.

**Comment by daveclayton** [comment]
HI, is this still required?

Happy to consider creating a branch here for that - it sounds logical.

**Comment by chb0github** [comment]
@daveclayton Well, anymore discussion about a a new release should be done on the forked repo as an issue and not here.

**Comment by jvitrifork** [comment]
did anything happen to this feature request?

**Comment by chb0github** [comment]
The project is dead and has been forked
